modules/apps/digital-signature/digital-signature-web/test/js/components/document-previewer/__snapshots__/DocumentPreviewerWrapper.js.snap

```diff
@@ -156,7 +156,7 @@ exports[`DocumentPreviewerWrapper renders with a document 1`] = `
       </li>
     </ul>
     <span>
-      document-1,1-of-x
```

Comment:
Why did these change?

Reply:
I'd call it an _improvement_.

@bryceosterhaus  If you compare the old mock to the new one, the old one was testing against a wrong implementation.

Just think about it, do we want `sub('document-x-of-x', [1, 1])` to display `document-1,1-of-x` or `document-1-of-1`?

The old test was simply written poorly.